VTEX - Catalog API.json

```diff
@@ -13598,7 +13598,7 @@
                         "name": "size",
                         "in": "query",
                         "required": true,
-                        "description": "Size of the itens in a page",
+                        "description": "Quantity of the itens in a page",
```

Comment:
Still prefer "Number of items in a page"

(note that "itens" is misspelled in this commit)

Reply:
Roger!